frontend/src/constants/packages/package.ts

```diff
@@ -0,0 +1,10 @@
+import { IconProps } from 'components';
+
+export type PackageType = {
```

Comment:
Rename file to `types.ts` for consistency

Reply:
done